### DISCUSSION TITLE
Room 1 - Series 3

### DISCUSSION THREAD

**Discussion by hanamayall** [IBAHCM/RPiR#214] [Archive]
Hana Mayall @hanamayall
Will McGhee @McGhee-Will
Dominik Vinopal @DominikVinopal

**Comment by hanamayall** [comment]
https://github.com/IBAHCM/hanamayallSeries03
https://github.com/IBAHCM/McGhee.Will.Series03
https://github.com/IBAHCM/DominikVinopalSeries03

**Comment by hanamayall** [comment]
New addition to the group!

Neema Zephania @neemazephania

**Reply by neemazephania** [reply]
repo - https://github.com/IBAHCM/neemazephaniaSeries3
